src/lib/seam/connect/routes/workspaces.ts

```diff
@@ -0,0 +1,41 @@
+import type { RouteRequestParams, RouteResponse } from '@seamapi/types/connect'
+import { Axios } from 'axios'
+import type { SetNonNullable } from 'type-fest'
+
+import { createAxiosClient } from 'lib/seam/connect/axios.js'
+import type { SeamHttpOptions } from 'lib/seam/connect/client-options.js'
+import { parseOptions } from 'lib/seam/connect/parse-options.js'
+
+export class WorkspacesHttp {
+  client: Axios
+
+  constructor(apiKeyOrOptionsOrClient: Axios | string | SeamHttpOptions) {
+    if (apiKeyOrOptionsOrClient instanceof Axios) {
+      this.client = apiKeyOrOptionsOrClient
+      return
+    }
+
+    const options = parseOptions(apiKeyOrOptionsOrClient)
+    this.client = createAxiosClient(options)
+  }
+
+  async get(
+    params: WorkspacesGetParams = {},
+  ): Promise<WorkspacesGetResponse['workspace']> {
+    const { data } = await this.client.get<WorkspacesGetResponse>(
+      '/workspaces/get',
+      {
+        params,
+      },
+    )
+    return data.workspace
+  }
+}
+
+export type WorkspacesGetParams = SetNonNullable<
+  Required<RouteRequestParams<'/workspaces/get'>>
+>
+
+export type WorkspacesGetResponse = SetNonNullable<
```

Comment:
if this alias is mainly for consumers to use as a convenience, wondering if a generic helper like `ApiResponse<"workspaces.get">` could work slightly better

might be personal preference but one of the things I don't like about our current SDK is that the top-level namespace is so cluttered

Reply:
After an offline discussion, we will remove them for now, but we may bring them back if we find they are useful for consumers.